nncf/quantization/algorithms/accuracy_control/algorithm.py

```diff
@@ -193,16 +200,33 @@ def apply(
             initial_metric, quantized_metric, self.max_drop, self.drop_type
         )
 
-        if should_terminate:
+        if False:
             QuantizationAccuracyRestorer._print_completion_message(accuracy_drop, self.drop_type)
             return quantized_model
 
         nncf_logger.info(f"Accuracy drop: {accuracy_drop} ({self.drop_type})")
 
-        if accuracy_drop <= self.max_drop:
-            return quantized_model
-
```

Comment:
This code was removed because it does not relevant now. The `calculate_accuracy_drop()` should be used.

Reply:
> Could you explain why?

This code snippet should be removed in #1859 PR because we already have the correct check here
https://github.com/openvinotoolkit/nncf/blob/d842fa8c85402ff71b1287078d96d332fe7788ee/nncf/quantization/algorithms/accuracy_control/algorithm.py#L196-L198

> Have we tests of stop conditions of algorithm?

There is the `test_calculate_accuracy_drop()` unit test.

> calculate_accuracy_drop used only under if self.tune_hyperparams_algorithm

It is not true. There are several places where this method is used. Please look again.